pkgs/tools/inputmethods/ibus-engines/ibus-libpinyin/default.nix

```diff
@@ -62,7 +62,7 @@ stdenv.mkDerivation rec {
     isIbusEngine = true;
     description = "IBus interface to the libpinyin input method";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ ericsagnes ];
+    maintainers = with maintainers; [ linsui ericsagnes ];
```

Comment:
I think you should add yourself in a different commit.

Reply:
Ok.